### ISSUE TITLE
Convert to keras-tf-pb with custom loss function

### ISSUE THREAD

**Issue by prnvjb** [Tony607/keras-tf-pb#2]
Hi,
I tried you model with a keras model of custom loss function but failed to convert as I'm getting the below error:
`FailedPreconditionError:` Attempting to use uninitialized value Variable
	 [[{{node _retval_Variable_0_5}} = _Retval[T=DT_FLOAT, index=5, `_device="/job:localhost/replica:0/task:0/device:CPU:0"](Variable)]]`

Thanks for your implementation.
